client/oci_env/utils.py

```diff
@@ -345,6 +346,8 @@ def compose_command(self, cmd, interactive=False, pipe_output=False):
         takes in the rest of the arguments (exec, up, down, etc) from the user.
         """
         binary = [self.config["COMPOSE_BINARY"] + "-compose", "-p", self.config["COMPOSE_PROJECT_NAME"]]
+        if sys.platform.lower() == "darwin" and self.config["COMPOSE_BINARY"] == "docker":
```

Comment:
```suggestion
        if self.config["COMPOSE_BINARY"] == "docker":
```
`docker-compose` has now been replaced with `docker compose`, so we should update it so.